Source/Scene/Scene.js

```diff
@@ -623,6 +631,25 @@ define([
             enabled : defaultValue(options.shadows, false)
         });
 
+        this._brdfLUT = new BrdfLutProcessor();
+
+        this._cubeMap = context.defaultCubeMap;
```

Comment:
The cube map stuff will stay because we are now using the star map instead of the horizon gradient. It is also going to be renamed from cubeMap to environmentMap for clarity.